tasks/linux.yml

```diff
@@ -56,12 +56,11 @@ tasks:
     cmds:
       - cp {{.ST_LINUXBOOT_KERNEL_CONFIG}} {{.KERNEL_CONFIG}}
       - cmd: >
-          if [ -n "{{.ST_LINUXBOOT_CMDLINE}}" ]; then
+          if [ -n '{{.ST_LINUXBOOT_CMDLINE}}' ]; then
           echo 'Overriding CONFIG_CMDLINE with
-          ST_LINUXBOOT_CMDLINE="{{.ST_LINUXBOOT_CMDLINE}}"';
-          sed -ie
-          's/CONFIG_CMDLINE=.*/CONFIG_CMDLINE="{{._ST_LINUXBOOT_CMDLINE}}"/g'
-          {{.KERNEL_CONFIG}};
+          ST_LINUXBOOT_CMDLINE="{{._ST_LINUXBOOT_CMDLINE}}"';
+          sed -ie '/CONFIG_CMDLINE=.*/d' {{.KERNEL_CONFIG}};
+          echo 'CONFIG_CMDLINE="{{._ST_LINUXBOOT_CMDLINE}}"' >> {{.KERNEL_CONFIG}};
```

Comment:
Has the cmdline something to do with the host configuration config thing, or do you fix something else here?

Reply:
This is to fix  #166. There was a regression, and parsing command-line with double quotes did not work anymore.